CODEOWNERS

```
@@ -1,51 +1,8 @@
-# Collection Stream
-/cmd/collection/** @jordanschalm
-/engine/collection/** @jordanschalm
-/protobuf/services/collection/** @jordanschalm
+# Protocol owners are not generally differentiated by sub-area for simplicity
+/** @jordanschalm @AlexHentschel @durkmurder @zhangchiqing @peterargue @Kay-Zee
 
-# Consensus Stream
-/cmd/consensus/** @AlexHentschel @durkmurder @jordanschalm
-/engine/consensus/** @AlexHentschel @durkmurder @jordanschalm
-
-# Execution Stream
-/cmd/execution/** @zhangchiqing
-/engine/execution/** @zhangchiqing
-
-# Access Stream
-/access/** @peterargue
-/cmd/access/** @peterargue
-/cmd/observer/** @peterargue
-/engine/access/** @peterargue
-
-# Verification Stream
-/cmd/verification/** @zhangchiqing
-/engine/verification/** @zhangchiqing
-/integration/tests/verification @zhangchiqing
-
-# Ledger Stream
-/ledger/** @AlexHentschel
-
-# FVM Stream
+# FVM specific
 /fvm/** @janezpodhostnik
```

```suggestion
/fvm/** @janezpodhostnik @zhangchiqing 
```

A PR author can't approve their own PR - I'm wondering if Janez makes a PR changing the FVM (and can't approve it) if it would be un-mergeable. Not sure if the CODEOWNERS protection handles this. Either way it's probably a good idea to have a backup.